samples/grids/tree-grid/data-searching/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample | Ignite UI | Web Components | infragistics</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" />
+    <link rel="stylesheet" href="/src/index.css" type="text/css" />
+
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample">         
+          <div class="container vertical">
+            <div style="margin-bottom: 1rem">
+              <igc-input id="searchBox" name="searchBox">          
+                <igc-icon id="clearIcon" slot="prefix" name="clear" collection="material"></igc-icon>
```

Comment:
Add checking which icon to show the search icon or clear icon based on whether there is a value entered in the igc-input.